app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsViewModel.kt

```diff
@@ -18,13 +19,19 @@ class TransactionDetailsViewModel
 
     fun loadDetails(txId: String) {
         safeLaunch {
-            val txDetails = transactionRepository.getTransactionDetails(txId)
-            updateState { TransactionDetailsViewState(txDetails, ViewAction.Loading(false)) }
+
+            kotlin.runCatching {
```

Comment:
This doesn't really make sense, as the `safeLaunch` has a catcher build in (https://github.com/gnosis/safe-android/blob/master/app/src/main/java/io/gnosis/safe/ui/base/BaseStateViewModel.kt#L50 , this can even be overwritten)

Reply:
I am not sure what to do here. 
Can I just remove the surrounding block and the error handling because we're logging it already in the base class?
The result qwould be this
```
    fun loadDetails(txId: String) {
        safeLaunch {
            val txDetails = transactionRepository.getTransactionDetails(txId)
            updateState { TransactionDetailsViewState(txDetails, ViewAction.Loading(false)) }
        }
    }
```